enhancements/split_helm_ansible_into_their_repos.md

```diff
@@ -0,0 +1,187 @@
+last-updated: 2020-08-30
+status: implementable
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Split Helm/Ansible into their projects
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+For Helm, besides the requirement of split the repo it also has a new pkg implementation which requires to be putted in place and is in the [joelanford/helm-operator](https://github.com/joelanford/helm-operator) currently. 
+
+This proposal is decoupling both requirements, which allow a specific strategy for each one of them. However, would this option be more comfortable to move forward with both needs or the approach to address them for Helm simultaneously could be more practical?
+
+**Note** A enhancement to describe the possible options to address the Helm requirement for the new pkg implementation ought to be made.
+
+## Summary
+
+This proposal has the goal to describe how we can split Helm/Ansible projects and then, decoupled from the CLI tool. 
+
+**Note** It is a follow up of the proposal [Split SDK into multiple projects](split-sdk-into-multiple-projects.md).   
+
+## Motivation
+
+- Improve the maintainability and reusability by adopting concepts such as [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) and [Single Responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle). 
+- Have healthier lifecycle since separately and individually, which means that we can have a single/granular releases.
+- Solve the most significant part of the problems faced with the CI. Currently, SDK repo takes to long to pass in the tests, which indeed is not just a problem to contribute to the projects but for us be able to cover more scenarios and ensure quality. 
+- Ensure that any type/language operator will be able to be built following the same base config and using a common lib which will be helpful to keep them maintained and supported as to provide more types of operators such as hybrids and done for other languages.  
+
+### Goals
+
+- The SDK CLI tool repository has just the code implementation, which is part of its domain of responsibility. 
+- A Helm Operator repository with all code implementation which is specific to it and is part of its domain of responsibility
+- An Ansible Operator repository with all code implementation which is specific to it and is part of its domain of responsibility
+- An SDK Operator Lib which contains the standard code that should be used to generate all types of Operators. (e.g. utils for releases which might be expected for all, templates and the implementation of the plugins which can be used for any language/type of operator)
```

Comment:
```suggestion
- Code shared by `operator-sdk`, `helm-operator`, and `ansible-operator` repos will be placed in the `operator-framework/operator-lib` repository when possible.
```

Reply:
operator-sdk should not use the operator-build-lib since it should no longer have code to build the projects and will just use the plugins for that. But the text was not good here. I changed let me know if it is better now.